### PR TITLE
Updating user password now changes tastypie API key

### PR DESCRIPTION
**Pull request by jrwdunham**

### CODE REVIEW

**Comment by Hwesta** [comment]
Add a note in the template informing the user that the API key will also be changed?  Otherwise :+1:

**Comment by jrwdunham** [comment]
Ok, @Hwesta, added the note in the template. Merge now?

File: storage_service/templates/administration/user_form.html [hunk 1]
```diff
@@ -19,6 +19,8 @@
     <form action="{% url 'user_edit' edit_user.id %}" method="post">
       {% csrf_token %}
       {{ password_form.as_p }}
+      <p>Note: if you change your password, your API key will also be changed
```
Comment (Hwesta) [inline storage_service/templates/administration/user_form.html]: Nitpick, but this can probably be one line

**Review by Hwesta — APPROVED** [review]
Looks great! :+1:

**Comment by jhsimpson** [comment]
:+1:

**Pull request merged**